resources_pages/install_ds_stack_windows.md

```diff
@@ -580,9 +580,7 @@ the right-most panel allows you to perform advanced modification
 of keyboards shortcuts in JupyterLab
 and it already contains quite a few shortcuts.
 We're going to add two more shortcuts,
-by pasting a text snippet just before the first existing shortcut.
-Go ahead and create a new line just after the line that says `"shortcuts": [`
-and paste the following:
+by pasting the following:
```

Comment:
Should we add the same change to the MacOS instructions? Was the keyboard shortcuts empty there also by default?